spec/models/cashbox_spec.rb

```diff
@@ -3,18 +3,28 @@
 RSpec.describe Cashbox, type: :model do
 
   describe '#total' do
-    it 'returns balance of cashbox' do
-      cashbox_source = Cashbox.create
-      cashbox_destination = Cashbox.create
+    context 'with existing payments' do
+      it 'returns balance of cashbox' do
+        cashbox_source = Cashbox.create
+        cashbox_destination = Cashbox.create
 
-      deposit_without_source = create(:payment, destination: cashbox_destination,
-                                      amount: 150)
-      deposit_with_source = create(:payment, destination: cashbox_destination,
-                                   source: cashbox_source, amount: 100)
-      withdrawal = create(:payment, amount: 50, source: cashbox_destination)
+        deposit_without_source = create(:payment, destination: cashbox_destination,
```

Comment:
Is there any way to use something like
```
let(:amount_1) { FactoryHelper::MySQL.int(min: 1, max: 2147483647) }
```
for these three amounts instead of hardocded integers?